internal/sql/driver/sql_service.go

```diff
@@ -49,7 +49,7 @@ type SQLService struct {
 	lg                   *logger.LogAdaptor
 }
 
-func newSQLService(cm *cluster.ConnectionManager, ss *iserialization.Service, fac *cluster.ConnectionInvocationFactory, is *invocation.Service, lg *logger.LogAdaptor) *SQLService {
+func NewSQLService(cm *cluster.ConnectionManager, ss *iserialization.Service, fac *cluster.ConnectionInvocationFactory, is *invocation.Service, lg *logger.LogAdaptor) *SQLService {
```

Comment:
We can also leave this newSQLService right? I just want to learn the motivation of the change.

Reply:
Right. Previously I created the SQLService in `hazelcast.Client`, so that function had to be exported, but there's no need to export it now.

Reply:
Since it's internal API, I left it as is.